sources/core/Stride.Core.Mathematics/Vector2.cs

```diff
@@ -210,19 +210,34 @@ public float[] ToArray()
         /// <summary>
         /// Moves the first vector2 to the second one in a straight line.
         /// </summary>
-        /// <param name="First">The first point.</param>
-        /// <param name="Second">The second point.</param>
-        /// <param name="MaxDistancePerCall">The rate at which the first point is going to move towards the second point.</param>
-        public static Vector2 Goto(Vector2 First, Vector2 Second, float MaxDistancePerCall)
+        /// <param name="from">The first point.</param>
+        /// <param name="to">The second point.</param>
+        /// <param name="maxDistancePerCall">The rate at which the first point is going to move towards the second point.</param>
+        /// <param name="result">When the method completes, contains a version of the Vector that is closer to the second vector.</param>
+        public static void Goto(ref Vector2 from, ref Vector2 to, ref float maxDistancePerCall, out Vector2 result)
```

Comment:
great, just one minor thing, single float values don't need to be passed by ref, only vectors. also, the name could be `maxTravelDistance`. as the "perCall" already implies a certain usage of the method.